src/main/java/com/powsybl/openloadflow/network/LfAction.java

```diff
@@ -54,58 +58,93 @@ public boolean isRelative() {
 
     private final TapPositionChange tapPositionChange;
 
-    private LfAction(String id, LfBranch disabledBranch, LfBranch enabledBranch, TapPositionChange tapPositionChange) {
+    private final Map<Pair<LfBus, String>, Pair<Double, PowerShift>> busesLoadShift; // key: bus and loadId.
```

Comment:
That would probably worth a dedicated objet, this are too many Pair which are difficult to read. You don't really need to map as you never search into it.